trackintel/analysis/tracking_quality.py

```diff
@@ -100,17 +101,17 @@ def temporal_tracking_quality(source, granularity="all"):
 
     elif granularity == "weekday":
         # get the tracked week relative to the first day
-        start_date = df["started_at"].min().date()
-        df["week"] = df["started_at"].apply(lambda x: (x.date() - start_date).days // 7)
+        start_date = df["started_at"].min().floor(freq="D")
+        df["week"] = ((df["started_at"] - start_date) // 7).dt.days
```

Comment:
Does the change keep the original behavior? i.e., shouldn't it be applying `.dt.days` first and then `//7`?